terminator/rectangler/main.go

```diff
@@ -71,12 +72,20 @@ func render(req api.RenderRequest) (api.RenderReply, error) {
 	}
 
 	output, err := rectangler(data, input.Bounds().Size())
+	_ = output
 	if err != nil {
 		return api.RenderReply{}, errors.Wrap(err, "rectangling")
 	}
 
+	modifiedPic, err := png.Decode(bytes.NewReader(req.Image))
+	_ = modifiedPic
+	if err != nil {
+		return api.RenderReply{}, errors.Wrap(err, "decoding image")
+	}
+
+	merged := merge(modifiedPic, output)
```

Comment:
maybe instead of modifiedPic and output, image and overlay? background and overlay? input and overlay?

This service doesn't know whether the input has been modified or not, and the name led me to briefly look for where we had modified it here.